charts/opensearch/templates/statefulset.yaml

```diff
@@ -224,8 +226,10 @@ spec:
         command: ['sh', '-c']
         args:
           - 'chown -R 1000:1000 /usr/share/opensearch/data'
+        {{- if not .Values.disableSecurityContexts }}
         securityContext:
           runAsUser: 0
+        {{- end }}
```

Comment:
WOuld it be better to port this into the base `values.yaml` and allow a user to overwrite with `securityContext: {}`

Reply:
Overwriting won't work. See https://github.com/helm/helm/issues/9136